static/sass/styles.scss

```diff
@@ -314,9 +314,9 @@ summary {
 }
 
 .organisations-hero {
-  background-image: url("#{$assets-path}214648ba-_M8A2368-final.jpg");
+  background-image: url("#{$assets-path}32d95e39-hero-laptop-org.jpg?w=2000");
```

Comment:
This seems a bit large for the use. About 1/2 the pixels still looks good to me

Reply:
ok, I already dropped it a bunch.